sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -21,6 +21,9 @@ namespace Azure.Messaging.ServiceBus
     /// The event handler is specified with the <see cref="ProcessMessageAsync"/>
     /// property. The error handler is specified with the <see cref="ProcessErrorAsync"/> property.
     /// To start processing after the handlers have been specified, call <see cref="StartProcessingAsync"/>.
+    ///
```

Comment:
Can we update the first sentence (line 18) to be:
The ServiceBusProcessor provides an abstraction around a set of receivers that allows using an event based model for processing received messages.

I think we can then remove lines 25-26.